packages/platform/platform-fastify/test/test-auth.integration.spec.ts

```
@@ -0,0 +1,345 @@
+    if (options && options.role === "admin" && request.get("authorization") !== "admin_token") {
+      throw new Forbidden("Forbidden");
+    }
+
+    ctx.getRequest().user = {
+      id: "id",
+      name: "name"
+    };
+  }
+}
+
+export function OAuth(options: any = {}): Function {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)